extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -0,0 +1,429 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# AMP A4A AD CREATIVE FORMAT
+
+** >> WORK IN PROGRESS.  SUBJECT TO CHANGE. << **
+
+_This set of standards is still in development and is likely to be revised.
+Feedback from the community is welcome.  Please comment here or on the [Intent
+to Implement](https://github.com/ampproject/amphtml/issues/4264)_.
+
+A4A (AMP Ads for AMPHTML Pages) is a mechanism for rendering fast, performant
+ads in AMP pages.  To ensure that A4A ad documents ("A4A creatives") can be
+rendered quickly and smoothly in the browser and do not degrade user experience,
+A4A creatives must obey a set of validation rules.  Similar in spirit to the
+[AMP format rules](../../spec/amp-html-format.md), A4A creatives have access to
+a limited set of allowed tags, capabilities, and extensions.
+
+## A4A Format Rules
+ 
+1. Unless otherwise specified below, the creative must obey all rules given by 
+the [AMP format rules](../../spec/amp-html-format.md), included here by 
+reference.  For example, the
+A4A [Boilerplate](amp-a4a-format.md#2) 
+deviates from the AMP standard boilerplate.
+
+  _*In addition*_:
+
+1. The creative must use `<html a4⚡>` or `<html a4amp>` as its enclosing tags.
+
+   _Rationale_: Allows validators to identify a creative document as either a 
+ general AMP doc or a restricted A4A doc and to dispatch appropriately.
+
+1. Media: Videos must not enable autoplay.
```

Comment:
Good questions.  I'm not sure what the edges of this should be, but the rationale of "don't force loading of video content" applies to all of them.  I'll add them all for now and we'll see if there's a strong counter-case.